docs/eog/case-studies-cw20-infrastructure.rst

```diff
@@ -0,0 +1,67 @@
+.. _CW20-Infrastructure: 
+
+CW20 Infrastructure
+====================
+
+We had already organised registration for the in-person event through `Eventbrite <https://www.eventbrite.com/>`_. 
+We sent emails to registrants through Eventbrite to notify them of our decision making every step of the way. 
+Because of the unconference nature of the event, we even asked for participant input to our contingency investigation. 
```

Comment:
```suggestion
Because of the unconference nature of the event, we even asked for participant input to our contingency investigation; surveying them to see if they preferred the event to stay as an in person event, move to online or be cancelled. 
```